cmake/stm32/g4.cmake

```diff
@@ -1,14 +1,18 @@
 set(STM32_G4_TYPES 
     G431xx G441xx G471xx G473xx G483xx G474xx G484xx
+    G491xx G4A1xx
 )
 set(STM32_G4_TYPE_MATCH 
     "G431.." "G441.." "G471.." "G473.." "G483.." "G474.." "G484.."
+    "G491.." "G4A1.."
 )
 set(STM32_G4_RAM_SIZES 
      32K  32K 128K 128K 128K 128K 128K
+    112K 112K

```

Comment:
IAR template file in STM32G4_HAL gives 128K for those two which is wrong! omg

Reply:
I opened an [issue](https://github.com/STMicroelectronics/STM32CubeG4/issues/19) to ST. Good catch